paw-jieba.el

```diff
@@ -76,10 +98,10 @@
           (let* ((buffer-read-only nil)
 		 (result (paw-chinese-format-sdcv-definitions word))
```

Comment:
When user calls paw-view-note,  `(funcall paw-search-function word buffer)` will be called, then `paw-sdcv-translate-result-async` will be called to search the word in a sub-process, when result comes back, we update the paw-view-note buffer inside `paw-sdcv-process-sentinel` function. So, to make it adapt on chinese, you may adpat to the same flow. Here we wait the result from `paw-chinese-format-sdcv-definitions` which blocks emacs.